website/package.json

```diff
@@ -21,21 +21,26 @@
   "dependencies": {
     "@docusaurus/core": "^2.0.0-alpha.72",
     "@docusaurus/plugin-content-pages": "^2.0.0-alpha.72",
-    "@docusaurus/theme-classic": "^2.0.0-alpha.72",
+    "@docusaurus/preset-classic": "^2.0.1",
```

Comment:
nit: it's better to keep the same version for all `@docusaurus/xxx` packge

Reply:
Got it, got it. I realized that navigation header would work as expected if I don't use the "preset-classic" theme. Next time, I will make sure to survey carefully the existing code structure before making changes.